[2022-07-05 09:32:52,872] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: sql_branch_dag.branch_sql scheduled__2022-07-04T00:00:00+00:00 [queued]>
[2022-07-05 09:32:52,905] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: sql_branch_dag.branch_sql scheduled__2022-07-04T00:00:00+00:00 [queued]>
[2022-07-05 09:32:52,905] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-07-05 09:32:52,905] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-07-05 09:32:52,905] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-07-05 09:32:52,945] {taskinstance.py:1377} INFO - Executing <Task(BranchSQLOperator): branch_sql> on 2022-07-04 00:00:00+00:00
[2022-07-05 09:32:52,953] {standard_task_runner.py:52} INFO - Started process 544 to run task
[2022-07-05 09:32:52,963] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'sql_branch_dag', 'branch_sql', 'scheduled__2022-07-04T00:00:00+00:00', '--job-id', '128', '--raw', '--subdir', 'DAGS_FOLDER/sql_branch_operator.py', '--cfg-path', '/tmp/tmp1zburupc', '--error-file', '/tmp/tmpl5or741d']
[2022-07-05 09:32:52,964] {standard_task_runner.py:80} INFO - Job 128: Subtask branch_sql
[2022-07-05 09:32:53,021] {task_command.py:370} INFO - Running <TaskInstance: sql_branch_dag.branch_sql scheduled__2022-07-04T00:00:00+00:00 [running]> on host dfb99dfc9d64
[2022-07-05 09:32:53,106] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=sql_branch_dag
AIRFLOW_CTX_TASK_ID=branch_sql
AIRFLOW_CTX_EXECUTION_DATE=2022-07-04T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-04T00:00:00+00:00
[2022-07-05 09:32:53,108] {sql.py:513} INFO - Executing: SELECT count(1) FROM user (with parameters None) with connection: postgres_con
[2022-07-05 09:32:53,117] {base.py:68} INFO - Using connection ID 'postgres_con' for task execution.
[2022-07-05 09:32:53,223] {base.py:68} INFO - Using connection ID 'postgres_con' for task execution.
[2022-07-05 09:32:53,228] {sql.py:531} INFO - Query returns 1, type '<class 'int'>'
[2022-07-05 09:32:53,229] {skipmixin.py:140} INFO - Following branch user_exist
[2022-07-05 09:32:53,230] {skipmixin.py:175} INFO - Skipping tasks ['user_not_exist']
[2022-07-05 09:32:53,273] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=sql_branch_dag, task_id=branch_sql, execution_date=20220704T000000, start_date=20220705T093252, end_date=20220705T093253
[2022-07-05 09:32:53,334] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-05 09:32:53,407] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
